docs/workflows.md

```diff
@@ -60,24 +62,25 @@ env:
 ```
 
 Update the values for your project and push a commit.
+If you used `cargo generate`, they should already be set up correctly.
```

Comment:
Nope this isn't true. `cargo generate` only sets up the project name and itch.io username, but there are other variables you may want to tune.

Reply:
Fair enough

Reply:
[`90c3b20` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/90c3b20de4a12bdd3e98b07dfd4c41793cccb7f2)